Packages/com.llealloo.audiolink/Runtime/Scripts/AudioLinkController.cs

```diff
@@ -109,6 +110,39 @@ ThemeColorController FindThemeColorController()
             return controllerTransform.GetComponent<ThemeColorController>();
         }
 
+        private void UpdateSyncMode(Transform inputTransform, ControllerSyncMode userSyncMode, ControllerSyncMode desiredSyncMode)
+        {
+
+            #if UDONSHARP
+
+                inputTransform.GetComponent<UdonBehaviour>().enabled = (int)userSyncMode < (int)desiredSyncMode;
+
+            #endif
+
+        }
+
+        public void SetControllerSyncMode(ControllerSyncMode syncMode)
```

Comment:
This seems to be missing some elements, such as everything relating to theme color.

Reply:
Included ThemeColorController in other toggles sync, lmk if there's something else I forgot to include